packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -1,18 +1,139 @@
+import { ECIES, PhoneNumberUtils, SignatureUtils } from '@celo/utils'
 import BigNumber from 'bignumber.js'
+import { zip } from 'lodash'
+import * as Web3Utils from 'web3-utils'
 import { Attestations } from '../generated/types/Attestations'
-import { BaseWrapper, proxySend } from './BaseWrapper'
+import { BaseWrapper, proxyCall, proxySend, toNumber, tupleParser, wrapSend } from './BaseWrapper'
+import { CeloToken } from './TokenWrapper'
+const parseSignature = SignatureUtils.parseSignature
 
 export interface AttestationStat {
   completed: number
   total: number
 }
 
+export interface IssuerAttestationState {
+  [issuer: string]: {}
+}
+
+export enum AttestationState {
+  None,
+  Incomplete,
+  Complete,
+}
+
+export interface ActionableAttestation {
+  issuer: string
+  attestationState: AttestationState
+  time: number
+  publicKey: string
+}
+
+const parseAttestationInfo = (rawState: { 0: string; 1: string }) => ({
+  attestationState: parseInt(rawState[0], 10),
+  time: parseInt(rawState[1], 10),
+})
+
+function attestationMessageToSign(phoneHash: string, account: string) {
+  const messageHash: string = Web3Utils.soliditySha3(
+    { type: 'bytes32', value: phoneHash },
+    { type: 'address', value: account }
+  )
+  return messageHash
+}
+
 export class AttestationsWrapper extends BaseWrapper<Attestations> {
+  attestationExpirySeconds = proxyCall(
+    this.contract.methods.attestationExpirySeconds,
+    undefined,
+    toNumber
+  )
+
+  getAttestationStat = proxyCall(
+    this.contract.methods.getAttestationStats,
+    tupleParser(PhoneNumberUtils.getPhoneHash, (x: string) => x),
```

Comment:
use `identity` parser here

Reply:
Typescript complained here, I couldn't figure out how to make identity work